proposals/wg-freebsd-runtime.md

```diff
@@ -0,0 +1,78 @@
+# Working Group Proposal: FreeBSD Runtime
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+# FreeBSD Runtime OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+With two working OCI runtimes for FreeBSD
+([runj](https://github.com/samuelkarp/runj),
+[ocijail](https://github.com/dfr/ocijail)) and several container engines
+including containerd, podman and cri-o, there is a need to define a
+FreeBSD-specific section of the runtime-spec to allow support for platform
+features such as resource limits and fine-grained jail permissions. This will
+help to ensure runtime compatibility and build a consensus for the best way to
+support FreeBSD container runtimes.
+
+## Scope
+
+* Define a FreeBSD extension to the runtime-spec similar in scope to the
+  existing extensions for linux, solaris, windows etc.
+  * Identify platform features which should be part of the FreeBSD spec.
+  * Build consensus on how to represent these features clearly.
+
+## Out of Scope
+
+* Identified out of scope items will be listed here as WG progresses.
```

Comment:
I think this needs to be fleshed out during the WG proposal process. The purpose of the "out of scope" section is to outline what is off the table for the WG to address. This section changing after the WG has been set up seems unlikely to happen (it requires a 2/3 TOB vote and if the WG doesn't want to do something, they can just choose not to do it) and having nothing here feels like a blank cheque.

Possible things that could be in "out of scope":

 * Changes to non-FreeBSD platform-specific sections.
 * The addition of any new FreeBSD-specific features or support to the "generic" sections of `config.json`.

Of course, it's up to you whether you feel these are appropriate or not.

Reply:
These are good suggestions.

Reply:
I like both suggestions, although I would like to list any 'pseudo' mount options such as tmpcopyup that we agree on in the Mounts section.